site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
+
+Chrome 91 adds the following features to the extensions platform.
+
+#### ES modules for service workers
+
+JavaScript supports modules in service workers. Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts. (https://www.chromestatus.com/guide/edit/4609574738853888)
+
+#### chrome.action.getUserSettings() available
+
+The new chrome.action.getUserSettings() API allows extensions to determine if the user has pinned the extension to the main toolbar.
+
+### Chrome 90 (stable on April 13, 2021)
+
+#### New chrome.scripting.removeCSS() method (MV3 only)
+
+The `chrome.scripting.removeCSS()` method lets extensions remove CSS that was previously inserted via
+chrome.scripting.insertCSS(). It replaces the MV2 method `chrome.tabs.removeCSS()`.
+
+#### chrome.scripting.executeScript() results now include frameId
+
+Results returned from `chrome.scripting.executeScript()` now include the frameId.  The frameId indicates the frame from which the result is from, allowing extensions to easily associate results with the individual frames when injecting in multiple frames.
+
+### April 8, 2021: Extension Manifest Converter
+
+The Chrome Extensions team has open sourced [Extension Manifest
+Converter](https://github.com/GoogleChromeLabs/extension-manifest-converter), a Python tool that
+helps to convert extensions to Manifest V3. 
+
+### Chrome 89 (stable on March 9, 2021)
+
+#### New chrome.tabGroups API
+
+Added the chrome.tabGroups API to let extensions read and manipulate tab groups. Manifest V3 only.
+
+#### Customizable permissions for MV3 Web Accessible Resources
+
+Manifest now supports permission definitions to restrict resources based on either the accessing site origin or extension ID.https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/
+
+
+### Chrome 88 (stable on January 19, 2021)
+
+#### Manifest V3 launched
+
+Manifest V3 launched in Chrome 88. This is a major update to the extensions platform; see Overview of Manifest V3 for a summary of new and changed features. Extensions may continue to use Manifest V2 for now, but this will be phased out in the near future. We strongly recommend that you use MV3 for any new extensions, and begin to migrate existing extensions to MV3 as soon as possible.
+
+
+
+
+
+{% if false %}
```

Comment:
Is the idea to actually hide old stuff? If so, please add a comment here.

Reply:
Can we just delete this content? The PR appears to completely replace it. If we really need to bring it back for some reason we can go back in git history.